src/components/file/file.tsx

```diff
@@ -95,6 +97,18 @@ export class File {
     @Prop({ reflect: true })
     public accept: string = '*';
 
+    /**
+     * Set to `false` to hide the visual representation of the dropzone.
+     */
+    @Prop()
+    public visualizeDropzone: boolean = true;
```

Comment:
Do we really need this option? Can't we at least wait to introduce it until someone asks for it?

Reply:
I think they already did when I demoed it. Example was when the component is used on a phone, they cannot drag and drop. So if someone wants to optionally hide this (like when they have a solution made for phones, or when god-knows-why) then they can do so programatically.

Reply:
it'll also prevent complains from people who don't like the change :P

Reply:
Ah… ok… hmm… 🤔

I rather think it would be better to build in feature-detection for drag-and-drop and not show this if drag-and-drop isn't supported. But that's more complexity that I also think we shouldn't do unless there's explicit demand for it.

I still suggest that we remove this feature until it's explicitly asked for. It's easy to add later, but harder to remove if we realise there's a better way to do it.

Second opinion @jgroth?

Reply:
Just "requesting changes" until we've decided this one (and I've looked over the css like Bef asked me to).

Reply:
Yeah I think I agree, it should be quite easy to add this later if it is needed :slightly_smiling_face: I general we should try and be a bit restrictive of what we add to the public interface since it always easier to add stuff than change or remove them

Reply:
OK. I'll remove it